### PR TITLE
Export import

### PR DESCRIPTION
**Pull request by dknochen**
frontend part

### CODE REVIEW

**Comment by dknochen** [comment]
@wcandillon you can already take a look if you want. 

the backend implementation is here : https://github.com/28msec/secxbrl.info/pull/492

this branch still needs an e2e test. Therefore, its still work in progress

**Comment by wcandillon** [comment]
@dknochen Yep that's the stuff. So fancy. I just miss the import from template part. And then I have some super minor comments, I just need to think about it a little more.

On the backend side, I asked a question which is unanswered and I also need to git it a little bit deeper before approving it.

**Comment by dknochen** [comment]
@wcandillon the template implementation is not part of this PR. Its a different feature and will be in a different PR. hope it makes sense.

**Comment by wcandillon** [comment]
@dknochen Looks good. Fancy stuff. Please remove the part where you can "rename the imported report". The rename feature is orthogonal. Other than that, it is good to merge.

File: app/modules/report-model.js [hunk 1]
```diff
@@ -11,7 +11,32 @@ angular
     ConceptIsStillReferencedError.prototype = new Error();
     return ConceptIsStillReferencedError;
 })
-.factory('Report', function($log, $q, ConceptIsStillReferencedError){
+.factory('ReportID', function(){
+
+    //Constructor
+    var ReportID = function(){
+        // thanks to https://gist.github.com/ae6rt/7894539
```
Comment (wcandillon) [inline app/modules/report-model.js:20]: It's not very clean to have this within the constructor.

**Comment by wcandillon** [comment]
Awesome

**Pull request merged**